nipype/utils/filemanip.py

```diff
@@ -29,6 +29,12 @@
 fmlogger = logging.getLogger("filemanip")
 
 
+related_filetype_sets = [
```

Comment:
I was imagining handling it later--but I'm also not so much of a stickler as to insist on changing this